shared/tsconfig.json

```diff
@@ -1,21 +1,16 @@
 {
-    "compilerOptions": {
-        "outDir": "./dist/",
-        "sourceMap": true,
-        "noImplicitAny": false,
-        "module": "commonjs",
-        "target": "es2017",
-        "jsx": "preserve",
-        "incremental": true,
-        "outDir": "./.tsOut/emit",
-        "tsBuildInfoFile": "./.tsOut/cache"
-    },
-    "include": [
-        "./**/*"
-    ],
-    "exclude": [
-        "node_modules",
-        "react-native",
-        "desktop/dist"
-    ]
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "noImplicitAny": false,
+    "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
Only added this, the rest is formatting. This is to support `import moment from 'moment'` and should match our reality better in general

Reply:
o cool! I had a different solution that used the `"allowSyntheticDefaultImports": true,` flag. I guess this is better?

Reply:
O hmm, I think we actually need this as well

Reply:
Yeah, I saw that one first too